### DISCUSSION TITLE
code on play.tailwindcss.com works, locally doesn't

### DISCUSSION THREAD

**Discussion by VitroidFPV** [tailwindlabs/tailwindcss#6830] [Help]
I'm just starting with Tailwind, playing around with effects I'm familiar with in normal CSS, and I'm stuck on one thing. On the play.tailwindcss.com site, this works as intended (expanding underline on hover) 
```html
<a class="bg-gradient-to-r from-blue-500 to-blue-400 bg-no-repeat bg-left-bottom bg-under-min hover:bg-under-full duration-300 cursor-pointer">About</a>
```
with the config 
```js
module.exports = {
  theme: {
    extend: {
      backgroundSize: {
        'under-min': '0% 3px',
        'under-full': '100% 3px',
      },
    },
  },
  plugins: [],
}
```
I have this exact same code running on a liveserver, but the effect no longer works. I'm not sure if I'm doing something wrong, or if this is a real issue, and if I'm missing something I'm sorry. I just want to get behind this, thanks for any help.

**Comment by adamwathan** [comment]
Hey! Looks like you need to configure your “content”:

https://tailwindcss.com/docs/content-configuration

**Reply by VitroidFPV** [reply]
I tried this, just including the index.html file, but it didn't work. Individually giving the `bg-under-min` and `bg-under-full` classes works to give it the underline, but on hover, the effect does not transition between them

**Reply by VitroidFPV** [reply]
I kinda bodged it together with JS `onmouseenter` and `onmouseleave`, and for now it kinda works
